flow-data/src/main/java/com/vaadin/flow/data/binder/Binder.java

```diff
@@ -1572,6 +1572,10 @@ private <T> T execute(Supplier<T> supplier) {
             } catch (BindingException exception) {
                 throw exception;
             } catch (Exception exception) {
+                if(binder == null) {
+                    throw new IllegalStateException(
+                            "This binding is already unbound", exception);
+                }    
```

Comment:
```suggestion
                if (binder == null) {
                    throw new IllegalStateException(
                            "This binding is already unbound", exception);
                }
```

Looks like Matti's formatter doesn't like his sofa ;)